src/deploydocs.jl

```diff
@@ -43,15 +43,23 @@ Otherwise the docs are deployed to the directory determined by the `devurl` argu
 
 The deployment procedure consists of the following steps:
 
-* Check out the `branch` of `repo` to a temporary location
-* Remove the existing deployment (sub-)directory with `git rm -r`
-* Copy the `target` (build) folder to the deployment directory
+* Check out the `branch` of `repo` to a temporary location.
+* Remove the existing deployment (sub-)directory with `git rm -r`.
+* Copy the `target` (build) folder to the deployment directory.
 * Generate `index.html`, and `versions.js` in the `branch` root and
-  `siteinfo.js` in the deployment directory
+  `siteinfo.js` in the deployment directory.
 * Add all files on the deployment `branch` (`git add -A .`), commit them, and
   push the `repo`. Note that any `.gitignore` files in the `target` directory
   affect which files will be committed to `branch`.
 
+!!! note
+    The `index.html` will be created at the `branch` root if and only if
+    one of the following two conditions hold:
+    (1) No such file already exists.
+    (2) The file exists and starts with the HTML comment
+    `<!--This file is automatically generated by Documenter.jl-->`.
```

Comment:
We could probably use a proper Markdown list here:

```suggestion

    1. No such file already exists.

    2. The file exists and starts with the HTML comment

        ```html
        <!--This file is automatically generated by Documenter.jl-->
        ```
```